components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/pom.xml

```diff
@@ -99,12 +99,6 @@
             <artifactId>jackson-databind</artifactId>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-web</artifactId>
-            <version>${springframework.spring-web.version}</version>
-            <scope>provided</scope>
```

Comment:
we should be able to remove the version definition of springframework.spring-web.version from parent pom

Reply:
https://github.com/wso2/carbon-identity-framework/pull/6153#pullrequestreview-2498028473